src/core/controllers/oidc.go

```diff
@@ -114,6 +116,7 @@ func (oc *OIDCController) Callback() {
 		oc.SendInternalServerError(err)
 		return
 	}
+	d.GroupIDs, err = group.GetGroupIDByGroupName(oidc.GroupsFromToken(idToken), common.OIDCGroupType)
```

Comment:
There is an err, it should be handled, or it will be overwrite by next line

Reply:
I'll just log this error but set empty list in the model.